app/settings.py

```diff
@@ -1,4 +1,5 @@
 import os
+import json
 
 
 SECURITY_USER_NAME = os.getenv('SEFT_CONSUMER_SECURITY_USER_NAME', 'dummy_user')
```

Comment:
I think that the os.getenv should either refer to a variable in the manifest of defined in vcap services ? Some of them here are not defined in cloudfoundry e.g SEFT_CONSUMER_HEALTHCHECK_DELAY

Reply:
Agree anything configurable in the settings.py needs to be in the manifest file. Then we can override it at deployment time.

in https://github.com/ONSdigital/sdx-seft-consumer-service/blob/master/app/settings.py